Makefile

```diff
@@ -115,6 +115,7 @@ deploy-with-olm:
 	sed -i 's#quay.io/metallb/metallb-operator-bundle-index:latest#$(BUNDLE_INDEX_IMG)#g' config/olm-install/install-resources.yaml
 	sed -i 's#mymetallb#$(NAMESPACE)#g' config/olm-install/install-resources.yaml
 	$(KUSTOMIZE) build config/olm-install | kubectl apply -f -
+	VERSION=$(VERSION) NAMESPACE=$(NAMESPACE) hack/wait-for-csv.sh
```

Comment:
can you explain why we need to wait as the tests are running with eventually and waiting for pods to come up before timeout.

Reply:
After some investigation we found out there's a pretty rare timing issue with the OLM lane, where the test finishes running in less than 30 seconds after the CRD created, resulting in the garbagecollector not doing it's job properly (has to do something with it not "discovering" the metallb crd in time - if we got it right). I added this check to buy some time for the cluster, and also it doesn't hurt to verify that the csv was installed properly without relying on the e2e-tests.